ci/pipeline.yml

```diff
@@ -86,7 +86,21 @@ jobs:
         args:
             - -exc
             - |
-              microdnf install tar gzip wget golang make findutils
+              microdnf install tar gzip wget make findutils
```

Comment:
**suggestion:** This should probably be `microdnf -y` (assume yes) to avoid interactive prompts:
> `-y, --assumeyes`
> Automatically answer yes for all questions.

https://dnf.readthedocs.io/en/latest/command_ref.html#options-label

```suggestion
              microdnf -y install tar gzip wget make findutils
```